github/compliance/model.py

```diff
@@ -37,6 +37,30 @@ def parse(value: int | str):
         return Severity(value)
 
 
+@dataclasses.dataclass(frozen=True)
+class MaxProcessingTimesDays:
+    '''
+    defines maximum processing time in days, based on issue "criticality"
```

Comment:
add hint about this dataclass being used for deserialisation from pipeline-definitions into docstr